src/app/styles/typography.scss

```diff
@@ -27,19 +27,19 @@ $fs-xxxl: 32px;
 
 @font-face {
   font-family: 'museo';
-  src: url('./assets/fonts/museo/museo-300.otf') format("opentype");
+  src: url('/assets/fonts/museo/museo-300.otf') format("opentype");
   font-weight: 300;
 }
 
 @font-face {
     font-family: 'museo';
-    src: url('./assets/fonts/museo/museo-500.otf') format("opentype");
+    src: url('/assets/fonts/museo/museo-500.otf') format("opentype");
```

Comment:
Prefer single quoted strings

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#stringquotes)